redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -5,7 +5,7 @@
  */
 final class rex_media_service
 {
-    public const ORDERBY = [
+    private const ORDER_BY = [
```

Comment:
haben wir die min-php version erhöht? Visibility auf const gibt es noch nicht sooo lange oder?

Reply:
Seit 7.1 schon. Ich komme in letzter Zeit aber auch öfters durcheinander, seit wann es welche PHP-Features gibt.